ncm-download/src/main/perl/download.pm

```diff
@@ -253,6 +254,7 @@ sub get_remote_timestamp
 sub retrieve
 {
     my ($self, $fn, %opts) = @_;
+    my $tmpfn = dirname($fn) . "/." . basename($fn) . ".part";
```

Comment:
Is it better to join file paths with the functionality provided by `File::Spec`?